app/services/idv/cancel_verification_attempt.rb

```diff
@@ -9,19 +9,11 @@ def initialize(user:)
     def call
       user.profiles.each do |profile|
         if profile.gpo_verification_pending?
-          profile.update!(
-            active: false,
-            deactivation_reason: :verification_cancelled,
-            gpo_verification_pending_at: nil,
-          )
+          profile.deactivate_for_verify_by_mail_cancelled
         end
 
         if profile.in_person_verification_pending?
-          profile.update!(
-            active: false,
```

Comment:
For both `profile.gpo_verification_pending?` and `profile.in_person_verification_pending?`, active was set to false on the profile. In the new methods, I don't see that you are updating the value of active. Is that something we need to do?